### PR TITLE
Associate Buf files with yaml

### PR DESCRIPTION
**Pull request by johanbrandhorst**
image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
 	"requires": true,
 	"packages": {
 		"": {
```
Comment (ZymoticB) [inline package-lock.json:7]: What's going on in here? Should this name be deleted? Should the version be `0.1.1`?

File: package.json [hunk 2]
```diff
@@ -53,7 +54,16 @@
 					"description": "Path to the buf binary"
 				}
 			}
-		}
+		},
+		"languages": [
```
Comment (bufdev) [inline package.json:58]: Does this do it just for the development of this plugin, or does this do it for users who install this plugin?

File: package-lock.json (hunk 1 above)
Comment (johanbrandhorst) [reply package-lock.json:7]: This file is managed by `npm`, I just ran `npm install` and it did this.

File: package.json (hunk 2 above)
Comment (johanbrandhorst) [reply package.json:58]: I tested this in the same way I tested the plugin during development, and it worked the way I wanted, so I am confident this is going to make this file association for any users using this extension.

File: package-lock.json (hunk 1 above)
Comment (ZymoticB) [reply package-lock.json:7]: 🤯

**Review by ZymoticB — APPROVED** [review]

**Review by bufdev — APPROVED** [review]

**Pull request merged**